boxsdk/object/item.py

```diff
@@ -613,3 +621,94 @@ def get_collaborations(self, limit=None, marker=None, fields=None):
+        }
+        template_key = 'securityClassification-6VMVochwUWo'
+        metadata_classification = self.metadata('enterprise', template_key).create(classification_metadata)
+        return metadata_classification['Box__Security__Classification__Key']
+
+    def update_classification(self, classification):
+        """
+        Updates metadata classification for the specified :class:`File` or :class:`Folder` object.
+
+        :param classification:
+            The classification to add to the :class:`File` or :class:`Folder`
+        :type classification:
+            `unicode`
+        :return:
+            The classification updated on the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        classification_metadata = self.metadata('enterprise', 'securityClassification-6VMVochwUWo')
+        updates = classification_metadata.start_update()
+        updates.update('/Box__Security__Classification__Key', classification)
+        metadata_classification = classification_metadata.update(updates)
+        return metadata_classification['Box__Security__Classification__Key']
+
+    def set_classification(self, classification):
+        """
+        Attempts to add a metadata classification to a :class:`File` or :class:`Folder`, if classification exists, then
+        do update.
+
+        :param classification:
+            The classification to add to the :class:`File` or :class:`Folder`
+        :type classification:
+            `unicode`
+        :return:
+            The classification set on the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        try:
+            metadata_classification = self.add_classification(classification)
+        except BoxAPIException as err:
+            if err.status == 409:
+                metadata_classification = self.update_classification(classification)
+            else:
+                raise
+        return metadata_classification
+
+    def get_classification(self):
+        """
+        Retrieves the classification specified for the :class:`File` or :class:`Folder`
+
+        :return:
+            The classification on the :class:`File` or :class:`Folder.
+        :rtype:
+            `unicode`
+        """
+        try:
+            classification = self.metadata('enterprise', 'securityClassification-6VMVochwUWo').get()
+        except BoxAPIException as err:
+            if err.status == 404:
+                return None
+            else:
+                raise
+        return classification['Box__Security__Classification__Key']
+
+    def remove_classification(self):
+        """
+        Removes a metadata classification from a :class:`File` or :class:`Folder`.
+
+        :returns:
+            Whether or not the delete was successful.
+        :rtype:
+            `bool`
+        """
+        return self.metadata('enterprise', 'securityClassification-6VMVochwUWo').delete()
```

Comment:
This will delete the entire template; we had discussed aligning with what the webapp does by deleting only the field containing the classification and leaving the template in place.

Also, how should we handle cases where there was already no classification?